nixos/modules/services/misc/gitlab.nix

```diff
@@ -752,8 +751,7 @@ in {
         description = ''
           The number of worker processes Puma should spawn. This
           controls the amount of parallel Ruby code can be
-          executed. GitLab recommends <quote>Number of CPU cores -
-          1</quote>, but at least two.
+          executed. GitLab recommends <literal>Number of CPU cores - 1</literal>, but at least two.
```

Comment:
This is weird. I did not realize the dash was supposed to be minus until reading the older version of GitLab installation requirements it appears to come from: https://docs.gitlab.com/13.12/ee/install/requirements.html

Not sure how to make it better, though.

Reply:
Maybe just reword it in a followup? Something like "one less than the number of CPU cores, but no less than two".

Reply:
BTW, the formula is more complicated nowadays: https://docs.gitlab.com/ee/install/requirements.html#puma-workers

Reply:
probably best to just refer to the documentation here instead of giving any concrete guidance. :/